bin/cron.js

```diff
@@ -101,6 +350,7 @@ cron.schedule("*/15 * * * *", async () => {
             apply: 0,
             reflection: 0,
             daily: 0,
+            inspirer: 0,
```

Comment:
I feel like this is scope creep of this function. Again, it's purely for updating the modules. Couldn't this be in the cron job that you made above?

Reply:
Sure yea can put it in the other